doc/versionHistory.rst

```diff
@@ -6,6 +6,13 @@
 Version History
 ##################
 
+.. _lsst.ts.ofc-2.0.2:
+
+v2.0.2
+======
+
+* ``OFC.get_correction()`` should give the aggregated degree of freedom (DOF).
+
```

Comment:
Can you please add an item to indicate that you removed out of date documents?

Reply:
Updated in the commit: e64d8ae. Thanks!